### ISSUE TITLE
Proposal: Allow overriding/disabling default TabView close tab hotkey

### ISSUE THREAD

**Issue by chingucoding** [microsoft/microsoft-ui-xaml#4930]
# Proposal:  Allow overriding/disabling default TabView close tab hotkey

## Summary

Not all apps benefit from using Ctrl+F4, some also register a second hotkey for closing a tab, e.g. Ctrl+W as this is also commonly used. Devs should be able to disable or replace the hotkey in case it is conflicting or there is a better suited alternative.

## Rationale

* Ctrl+F4 might already be in use inside the app
* Other hotkeys might be more familar for users or might make more sense

## Scope

| Capability | Priority |
| :---------- | :------- |
| Allow disabling the Ctrl+F4 hotkey | Must |
| Let devs provide an alternative hotkey easily | Should |

## Open Questions

@stmoy FIY

What about the TabVIiew tooltip, should that be changed easily too then?

**Comment by StephenLPeters** [comment]
Seems like a great idea.  @chigy and @beervoley FYI

**Comment by chingucoding** [comment]
I assume this would need to go through API review first? Or can we implement this as preview API and go from there?

**Comment by ZodmanPerth** [comment]
I would just like to turn off the shortcuts altogether.

**Comment by ZodmanPerth** [comment]
Further our app allows the user to specify their own language, which may be different from the Windows language.  If we can't override the ToolTip here (and on the scroll button) ourselves then it's possible the ToolTip "hardcoded" in the control will be in a different language than specified by our app, causing a UX issue.

**Comment by chingucoding** [comment]
How do you change the language of the app? If you change the PrimaryLanguageOverride, the ToolTip texts will also change the language accordingly (at least they should).

**Comment by ZodmanPerth** [comment]
Yes we use PrimaryLanguageOverride.  You're right; that will change the tooltip language, my bad.  Scratch that part of the issue.
So it's just a matter of being able to override/turn off the tooltip itself.
